src/pycram/knowledge/knowledge_sources/__init__.py

```diff
@@ -0,0 +1,7 @@
+from os.path import dirname, basename, isfile, join
+import glob
+
+modules = glob.glob(join(dirname(__file__), "*.py"))
+__all__ = [basename(f)[:-3] for f in modules if isfile(f) and not f.endswith('__init__.py')]
```

Comment:
this looks fishy, please explain

Reply:
This imports all files in this directory to ensure that all knowledge sources are known.